BridgeSDK/BridgeAPI/BridgeAPITypes/MachineGeneratedDoNotEdit/_SBBSurveyElement.m

```diff
@@ -42,6 +43,10 @@ @interface _SBBSurveyElement()
 
 @property (nonatomic, strong, readwrite) NSArray *afterRules;
```

Comment:
Um, shouldn't these be nullable?

Reply:
The way the templates work, these get initialized to empty arrays regardless of whether they contain any rules.